libr/core/cmd_write.c

```diff
@@ -1788,3 +1843,66 @@ static int cmd_write(void *data, const char *input) {
 	R_FREE (ostr);
 	return 0;
 }
+
+static void cmd_write_init(RCore *core, RCmdDesc *parent) {
+	DEFINE_CMD_DESCRIPTOR (core, w);
+	DEFINE_CMD_DESCRIPTOR (core, wa);
+	DEFINE_CMD_DESCRIPTOR (core, wA);
+	DEFINE_CMD_DESCRIPTOR (core, wc);
+	DEFINE_CMD_DESCRIPTOR (core, we);
+	DEFINE_CMD_DESCRIPTOR (core, wo);
+	DEFINE_CMD_DESCRIPTOR (core, wop);
+	DEFINE_CMD_DESCRIPTOR (core, wp);
+	DEFINE_CMD_DESCRIPTOR (core, wt);
+	DEFINE_CMD_DESCRIPTOR (core, wv);
+	DEFINE_CMD_DESCRIPTOR (core, wx);
+
+	RCmdDesc *wv_cd = r_cmd_desc_argv_new (core->rcmd, parent, "wv", wv_handler);
+	r_return_if_fail (wv_cd);
+	RCmdDescHelp wv_help;
```

Comment:
actually, instead of working on a brand new `RCmdDescHelp` and then having to write it with `r_cmd_desc_set_help` I could just get `cd->help` and directly work on it. What do you think?

Reply:
Do you mean setting fields inside `RCmdDesc->help` directly? I am fine with that.

Reply:
Sounds good to me as well.